sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -267,6 +267,7 @@ object SQLConf {
       "Note: For structured streaming, this configuration cannot be changed between query " +
       "restarts from the same checkpoint location.")
     .intConf
+//    .checkValue(_ > 0, "the value of sql.shuffle.partitions must be greater than 0")
```

Comment:
Uncomment?

Reply:
reproducing the bug

Reply:
the test reproducing the bug,

Reply:
admins verify this bug I will add checkValue or haven other idear(

Reply:
`Adding tests which reproduces a bug` meant adding tests which fail when we don't apply this patch and passes when we apply this patch. So if you confirmed it, you can uncomment and let tests pass.